src/Utility/StringUtil.ts

```diff
@@ -1,7 +1,11 @@
+import * as XRegExp from "xregexp";
 import {TypeUtil} from "./TypeUtil";
 import {InvalidOperationException} from "../Database/DatabaseExceptions";
 
 export class StringUtil {
+  private static readonly letterRe: RegExp = <RegExp>XRegExp("^\\p{L}$");
```

Comment:
How is this different from `[A-Za-z]`? Do we really need some external library for regexes - `xregexp`?

Reply:
Because in C# isLetter check is character is an letter of any alphabet available in Unicode (latin, cyrillic, arabic, hebrew etc) while a-zA-Z checks only latin. In default Javascript Regexp there are no funcitonality to check for any alphabet. I googled about it and found this library

Reply:
ok